.github/workflows/update-snapshots.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     steps:
       - name: Generate token
         id: generate_token
-        uses: tibdex/github-app-token@v1
+        uses: actions/create-github-app-token@v1
```

Comment:
Is this doing a similar job to @camertron's action? https://github.com/camertron/github-app-installation-auth-action

Reply:
Awesome, yeah we should def migrate to the official action 👍